client/app/scripts/superdesk-desks/desks.js

```diff
@@ -868,7 +867,7 @@
                         scope.desk.edit = _.create(desk);
                     };
 
-                    scope.save = function(desk) {
+                    scope.save = function(desk, done) {
```

Comment:
Docstring would be useful, explaining (among other things), for example,  what the `done` parameter represents.